.github/workflows/pipeline-workflow.yml

```diff
@@ -0,0 +1,145 @@
+name: CI/CD Pipeline
+
+on:
+  push:
+    branches: [ master, dev ]
+env:
+  tf_version: '0.12.26'
+
+jobs:
+
+  build_and_deploy:
+    name: Build and deploy Fargate API
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Setup DEV Environment Variables
+      if: github.ref == 'refs/heads/dev'
+      run: |
+        echo "::set-env name=aws_key::${{ secrets.byu_oit_terraform_dev_key }}"
+        echo "::set-env name=aws_secret::${{ secrets.byu_oit_terraform_dev_secret }}"
```

Comment:
Do we need to indicate here that some of these secrets need to be setup by the developer?

Reply:
I considered that. There are lots of pieces that need to be customized when this template is instantiated. I think I captured them all in README.md

Reply:
Or maybe I didn't do a great job with that.

Reply:
I wasn't really looking at it from the readme perspective, but I'm the kind of guy that never reads the instructions on things either.